pkg/test/framework/components/istio/operator.go

```diff
@@ -400,11 +400,22 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 			// before the external control plane cluster. Since remote clusters use gateway injection, we can't install the gateways
 			// until after the control plane is running, so we install them here. This is not really necessary for pure (non-config)
 			// remote clusters, but it's cleaner to just install gateways as a separate step for all remote clusters.
-			if err = installRemoteClusterGateways(s, i, c); err != nil {
-				return i, err
+
+			// Config cluster may have a separate config from pure remote clusters.
+			configFiles := []string{
+				filepath.Join(testenv.IstioSrc, IntegrationTestRemoteGatewaysIOP),
+			}
```

Comment:
nit: move the config file selection int `installRemoteClusterGateways` to avoid future usages passing the wrong file